Sources/Testing/Running/Runner.swift

```diff
@@ -188,30 +184,30 @@ extension Runner {
     defer {
       if let step = stepGraph.value {
         if shouldSendTestEnded {
-          Event.post(.testEnded, for: (step.test, nil), configuration: configuration)
+          Event.post(.testEnded, for: (step.test, nil), configuration: _configuration)
         }
-        Event.post(.planStepEnded(step), for: (step.test, nil), configuration: configuration)
+        Event.post(.planStepEnded(step), for: (step.test, nil), configuration: _configuration)
       }
     }
 
     if let step = stepGraph.value, case .run = step.action {
       await Test.withCurrent(step.test) {
-        _ = await Issue.withErrorRecording(at: step.test.sourceLocation, configuration: configuration) {
+        _ = await Issue.withErrorRecording(at: step.test.sourceLocation, configuration: _configuration) {
```

Comment:
Should we update `withErrorRecording` to just pull the `current` task-local value? Do we ever use it in a way that needs to take any other value?

Reply:
Given where I landed with other changes I don't think I'll need this right now

Reply:
But I agree in principle we could make that utility infer the current configuration, just not sure it's needed for this PR